add before you begin content for aws lambda (#5889)
getsentry/sentry-docs: src/platforms/node/guides/aws-lambda/index.mdx

</Note>

Before you begin, note:

- The estimated time to configure the integration is 5 - 15 minutes.

> - You should have experience with AWS console and a basic understanding of Lambda.

Author: ```suggestion
- You need to have experience with AWS console and a basic understanding of Lambda.
```

> - You must have an AWS account to create an [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html).

Author: ```suggestion
- You must have an AWS account to create an [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html).
```

> - You must have an AWS account to create an [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html).

Author: ```suggestion
- You'll need an AWS account and to create an [IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html).
```
